### DISCUSSION TITLE
Where can I find out more?  What are the caveats?  How can I interface with code in the browser (i.e. js methods?)

### DISCUSSION THREAD

**Discussion by catmando** [ruby/ruby.wasm#146] [Q&A]
Just bumped into this.  I am a long time Opal developer, and this looks great, but there has to be some gotchas.  

Has anybody used this for an actual browser app?

How do you debug?  Is there anyway to set a break point?

I assume based on the minimal examples that it includes a full compiler (i.e. full eval) Can that be excluded if you don't need it?

Etc. etc...

**Comment by kateinoigakukun** [comment]
I appreciate your interest! Yes, it's still in a very early stage and a lot of documentation is still missing. (Of course, contributions from the community is really welcome). Anyway, I'll try to enhance our documentation content next month

Here is some links to resources:
- https://github.com/ruby/ruby.wasm/wiki/Showcase
- https://ruby.github.io/ruby.wasm/
- https://github.com/ruby/ruby.wasm/tree/main/packages/npm-packages/ruby-head-wasm-wasi
